recirq/otoc/loschmidt/tilted_square_lattice/run-simulator.py

```diff
@@ -0,0 +1,24 @@
+import cirq
+from cirq_google.workflow import QuantumRuntimeConfiguration, \
+    SimulatedProcessorWithLocalDeviceRecord, execute
+from recirq.cirqflow.run_utils import get_unique_run_id
+from recirq.otoc.loschmidt.tilted_square_lattice import TiltedSquareLatticeLoschmidtSpec
+
+assert TiltedSquareLatticeLoschmidtSpec, 'register deserializer'
+
+FN = 'loschmidt.tilted_square_lattice.small-v1.json.gz'
```

Comment:
Same comments about FN and docstrings in this file as well.

Reply:
done